### DISCUSSION TITLE
Batch editor string for Technical Records legal moves

### DISCUSSION THREAD

**Discussion by KingMarzy** [kwsch/PKHeX#3641] [Q&A] [answered]
Is it possible to batch edit the "Give All" button for TR moves?

**Comment by kwsch** [comment]
`.RelearnMoves=$suggest`

**Reply by KingMarzy** [reply]
Tried that over and over but doesn't produce the desired outcome, it only gives 1-4 moves depending on what the current moves are.

I had a mon who i game all legal TR moves to and removed all then I ran .RelearnMoves=$suggest in batch editor. It only gives the 4 moves under "Current Moves"
![image](https://user-images.githubusercontent.com/63757067/204062814-58b485a5-b97f-4f08-9413-48706ed50d99.png)

**Comment by Lusamine** [comment, marked as answer]
Use `.RelearnMoves=$suggestAll`.

**Reply by KingMarzy** [reply]
That did it. Thanks!
